packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -399,10 +416,88 @@ const RealAccountSignup = ({
+        }
+    };
+
+    const handleRiskAcceptance = () => {
+        closeRealAccountSignup();
+        setShouldShowRiskWarningModal(false);
+    };
+
+    if (should_show_risk_warning_modal) {
+        return (
+            <RiskToleranceWarningModal
+                show_risk_modal={should_show_risk_warning_modal}
+                onClick={handleRiskAcceptance}
+                title={risk_warning_title}
+                body_content={
+                    <Localize
+                        i18n_default_text='CFDs and other financial instruments come with a high risk of losing money rapidly due to leverage. You should consider whether you understand how CFDs and other financial instruments work and whether you can afford to take the high risk of losing your money. <0/><0/> To continue, kindly note that you would need to wait 24 hours before you can proceed further.'
+                        components={[<br key={0} />]}
+                    />
+                }
+            />
+        );
+    } else if (should_show_appropriateness_warning_modal) {
+        return (
+            <TestWarningModal
+                show_risk_modal={should_show_appropriateness_warning_modal}
+                body_content={
+                    <Text as='p' size='xs'>
+                        <Localize
+                            i18n_default_text='In providing our services to you, we are required to ask you for some information to assess if a given product or service is appropriate for you and whether you have the experience and knowledge to understand the risks involved.<0/><0/>'
+                            components={[<br key={0} />]}
+                        />
+                        <Localize
+                            i18n_default_text='On the basis of the information provided in relation to your knowledge and experience, we consider that the investments available via this website are not appropriate for you.<0/><0/>'
+                            components={[<br key={0} />]}
+                        />
+                        <Localize i18n_default_text='By clicking ‘Accept’ and proceeding with the account opening, you should note that you may be exposing yourself to risks. These risks, which may be significant, include the risk of losing the entire sum invested, and you may not have the knowledge and experience to properly assess or mitigate them.' />
+                    </Text>
+                }
+                footer_content={
+                    <React.Fragment>
+                        <Button type='button' large text={localize('Decline')} secondary onClick={handleOnDecline} />
+                        <Button type='button' large text={localize('Accept')} primary onClick={handleOnAccept} />
+                    </React.Fragment>
+                }
+            />
+        );
+    }
     return (
         <React.Fragment>
-            <DesktopWrapper>
-                {is_real_acc_signup_on && (
+            {is_real_acc_signup_on && (
```

Comment:
Can we wrap both the `DesktopWrapper` and `MobileWrapper` in the `is_real_acc_signup_on` condition?

Reply:
Yes we can. Will fix this